text_volpiano_alignment.py

```diff
@@ -100,13 +100,14 @@ def _align_word(text_word: "list[str]", volpiano_word: str) -> "list[tuple[str,
     # Squash final syllables of a word together if there are more
     # syllables in the text than in the volpiano.
     if len(text_word) > len(vol_syls):
```

Comment:
huh - right, this is a proper parameter for the function. Yeah, I think I'd prefer `def _align_word(text_syls: "list[str]", volpiano_word: str)` - this way, the names of the parameters would provide a hint that they take different types, even before looking at the type annotations. `text_syls` is in the plural, so it should be a list of multiple syllables, while `volpiano_word` is in the singular, so it should be a single string.

Reply:
...also, what's going on with the type annotation? `text_word: "list[str]"` - you're annotating it as a string, `"list[str]"`... does this work? Suggest you use `from typing import List`; `text_syls: List[str]`

Reply:
> does this work?

Yes, this is what pylance tells me to do. But looking it up (which, shame on me for not doing that til now), it seems that this is only PEP standard for forward references. Since this is a thing the whole codebase over, leaving this until #10